utils/upload_utils.py

```diff
@@ -58,25 +56,29 @@ def post_texts(st, stub, userDataObject, text_list, metadata_list):
 
         inputs = []
         for idx, text in enumerate(text_batch):
+            input_obj = Inputs(logger_level="ERROR")
```

Comment:
Could be initialised at the beginning of func with user_id and app_id, not required for every loop